app/models/metasploit/credential/nonreplayable_hash.rb

```diff
@@ -39,88 +92,193 @@ class Metasploit::Credential::NonreplayableHash < Metasploit::Credential::Passwo
     hmac-sha384
     hmac-sha512
     hmailserver
+    hsrp
+    ike
     ipb2
+    itunes-backup
+    iwork
     keepass
     keychain
+    keyring
+    keystore
+    known_hosts
     krb4
     krb5
+    krb5-17
+    krb5-18
+    krb5-3
+    krb5asrep
+    krb5pa-md5
+    krb5pa-sha1
+    krb5tgs
+    kwallet
+    lastpass
+    leet
     lm
     lotus5
-    md4-gen
-    md5
-    md5crypt-opencl
+    lotus85
+    lp
+    lpcli
+    luks
+    md2
+    md5crypt
+    md5crypt-long
     md5ns
+    mdc2
     mediawiki
+    monero
+    money
+    mongodb
+    mozilla
     mscash
     mscash2
-    mscash2-opencl
     mschapv2
-    mskrb5
+    mschapv2-naive
     mssql
     mssql05
+    mssql12
+    multibit
     mysql
     mysql-sha1
-    mysql-sha1-opencl
+    mysqlna
+    net-ah
+    net-md5
+    net-sha1
     nethalflm
     netlm
     netlmv2
     netntlm
+    netntlm-naive
     netntlmv2
-    nsldap
+    nk
+    notes
+    nsec3
     nt
-    nt-opencl
-    nt2
+    o10glogon
+    o3logon
+    o5logon
     odf
```

Comment:
It looks like some of the `opencl` entries have been removed from this list. Checking framework it looks like that would only impact this line:

https://github.com/rapid7/metasploit-framework/commit/6d2b7e05ef09c0d84f731b2e7cef3f1814b4a8c4

But that jtr_format never existed here 🤔

Reply:
This list is really only used for [search terms from](https://github.com/rapid7/metasploit-framework/blob/950e976f7b288bc681e5c30cbb465dff1e0a5505/lib/msf/ui/console/command_dispatcher/creds.rb#L406-L412) what I can tell, since the value is rare I can see omitting it.